pylinkvalidator/models.py

```diff
@@ -207,6 +214,9 @@ def _parse_config(self):
         if self.options.ignored_prefixes:
             self.ignored_prefixes = self.options.ignored_prefixes.split(',')
 
+        if self.options.excluded_urls:
+            self.excluded_urls = self.options.excluded_urls.split(',')
```

Comment:
slight optimization to prevent compiling the regex for every url:

```python
self.excluded_urls = [re.compile(pattern) for pattern in self.options.excluded_urls.split(',')]
```